apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_utils/detectedFeatures/claimConditions.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  isGetActiveClaimConditionIdSupported as isGetActiveClaimConditionIdSupported_ERC20,
+  isGetActiveClaimConditionSupported as isGetActiveClaimConditionSupported_ERC20,
+  isGetClaimConditionByIdSupported as isGetClaimConditionByIdSupported_ERC20,
+  isGetClaimConditionsSupported as isGetClaimConditionsSupported_ERC20,
+  isResetClaimEligibilitySupported as isResetClaimEligibilitySupported_ERC20,
+  isSetClaimConditionsSupported as isSetClaimConditionsSupported_ERC20,
+} from "thirdweb/extensions/erc20";
```

Comment:
just do `import * as ERC20Ext from "..."` and then access it off of that, it still gets tree shaken away properly

Reply:
updated